feature-group-definitions/check-visibility.yml

```
@@ -1,4 +1,5 @@
 name: checkVisibility()
+description: "The `Element.checkVisibility()` method checks if an element is visible. For example, it checks whether an element has the style `display: none`."
```

I think we should avoid the `Interface.method()` notation unless it's about a static method like `CSS.supports()`, since it looks like code but won't work. Sometimes I think it's OK to use a variable name that is very likely to be used, like `element. checkVisibility()` in this case, but let's see if it can be avoided:

```suggestion
description: "The `checkVisibility()` method checks if an element is visible. For example, it checks whether an element has the style `display: none`."
```

In cases where we have to spell out the interface name I think one of these styles would work:

```suggestion
description: "`Element`'s `checkVisibility()` method checks if an element is visible. For example, it checks whether an element has the style `display: none`."
```

```suggestion
description: "The `checkVisibility()` method for `Element` checks if an element is visible. For example, it checks whether an element has the style `display: none`."
```